src/simpleitk_timeseries_motion_correction/motion.py

```
@@ -266,97 +271,133 @@ def register_pair(
+    initial_transform.SetCenter(com_initializer.GetCenter())
+
+    registration_method.SetInitialTransform(initial_transform, inPlace=False)
+
+    # Execute registration
+    # If the registration fails, return the identity transform
+    try:
+        final_transform = registration_method.Execute(
+            sitk.Cast(fixed_slice, sitk.sitkFloat32),
+            sitk.Cast(moving_slice, sitk.sitkFloat32),
+        ).GetBackTransform()
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)